### PR TITLE
Fixes #28991 - Add description to plugin roles

### PR DESCRIPTION
**Pull request by stejskalleos**
PR is part of  [Add default descriptions to all roles added from plugins](https://projects.theforeman.org/issues/28936) task.

### CODE REVIEW

**Comment by stejskalleos** [comment]
Hi @timogoebel , can you please give me some good description for the roles? Thanks

File: lib/puppetdb_foreman/engine.rb [hunk 1]
```diff
@@ -33,8 +33,9 @@ class Engine < ::Rails::Engine
                                              :'api/v2/puppetdb_nodes' => [:import]
         end
 
-        role 'PuppetDB Node Viewer', [:view_puppetdb_nodes]
-        role 'PuppetDB Node Manager', [:view_puppetdb_nodes, :destroy_puppetdb_nodes, :import_puppetdb_nodes]
+        role 'PuppetDB Node Viewer', [:view_puppetdb_nodes], 'Role granting permissions to view Puppet nodes.'
```
Comment (timogoebel) [inline lib/puppetdb_foreman/engine.rb]:
```suggestion
        role 'PuppetDB Node Viewer', [:view_puppetdb_nodes], 'Role granting permissions to view nodes from PuppetDB.'
```

File: lib/puppetdb_foreman/engine.rb [hunk 2]
```diff
@@ -33,8 +33,9 @@ class Engine < ::Rails::Engine
...
+        role 'PuppetDB Node Manager', [:view_puppetdb_nodes, :destroy_puppetdb_nodes, :import_puppetdb_nodes], 
+             'Role granting permissions to manage Puppet nodes.'
```
Comment (timogoebel) [inline lib/puppetdb_foreman/engine.rb]:
```suggestion
             'Role granting permissions to manage nodes from PuppetDB.'
```

**Comment by stejskalleos** [comment]
[test puppetdb_foreman]

**Pull request closed without merging**